src/DataFixtures/UserFixtures.php

```diff
@@ -15,13 +15,53 @@ public function load(ObjectManager $manager): void
     {
         $faker = Factory::create('fr_FR');
 
+        // création admin
+        $user = new User();
```

Comment:
Dans ce fichier tu redéfinie plusieurs la même variable $user. 
Il faudrait plus faire : 
```suggestion
        $admin = new User();
        $admin->setEmail($faker->email());
```

Reply:
changements faits!